hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -106,6 +118,14 @@ public LocalRegionCache(final String name, final HazelcastInstance hazelcastInst
         }
     }
 
+    private static MapConfig lookupMapConfig(String name, HazelcastInstance hazelcastInstance) {
+        try {
+            return hazelcastInstance != null ? hazelcastInstance.getConfig().findMapConfig(name) : null;
+        } catch (UnsupportedOperationException ignored) {
+            return null;
+        }
+    }
```

Comment:
This method contains the logic from the old constructor, extracted out as a static method for use by the existing constructor.